sdk/python/feast/infra/feature_servers/multicloud/Dockerfile.dev

```diff
@@ -1,24 +1,28 @@
-FROM debian:11-slim
+FROM python:3.11-slim-bullseye
 
-RUN apt update && \
-        apt install -y \
-        jq \
-        python3 \
-        python3-pip \
-        python3-dev \
-        build-essential
+RUN pip install --no-cache-dir pip --upgrade
+RUN pip install --no-cache-dir pip-tools
 
-RUN pip install pip --upgrade
-COPY . .
+RUN apt update && apt install -y -V ca-certificates lsb-release wget make git curl gcc && \
+    curl -sL https://deb.nodesource.com/setup_20.x | bash - && \
+    wget https://apache.jfrog.io/artifactory/arrow/$(lsb_release --id --short | tr 'A-Z' 'a-z')/apache-arrow-apt-source-latest-$(lsb_release --codename --short).deb && \
```

Comment:
one of these days we should investigate on the reason why these arrow source code bundle for debian is installed.
AFAIK, installing the pyarrow python package is the only thing we need to connect an arrow system.
I will track a GH issue to investigate and in case clear the dockerfile.

Reply:
https://github.com/feast-dev/feast/issues/4785